pkg/sql/sem/tree/eval.go

```diff
@@ -2172,9 +2172,13 @@ type EvalContext struct {
 	// of a transaction. Used for now(), current_timestamp(),
 	// transaction_timestamp() and the like.
 	TxnTimestamp time.Time
-	// The cluster timestamp. Needs to be stable for the lifetime of the
-	// transaction. Used for cluster_logical_timestamp().
-	ClusterTimestamp hlc.Timestamp
```

Comment:
Making this not exported will break https://github.com/cockroachdb/cockroach/pull/22291 because the distsql serializer uses the raw value since it needs to skip the not-zero assertion.